packages/node/src/handlers.ts

```diff
@@ -42,13 +43,15 @@ export function tracingHandler(): (
       if (span) {
         traceId = span.traceId;
         parentSpanId = span.parentSpanId;
+        sampled = span.sampled;
```

Comment:
This is a bug fix right? We should remember for the changelog then that there are two changes in this PR. a `feat` for the meta tag, and a `fix` for sampling from trace parent in node.